website/docs/taskfile_versions.mdx

```diff
@@ -60,6 +60,13 @@ Now when someone tries to run the Taskfile with an older version of Task, they
 will receive an error prompting them to upgrade their version of Task to
 `v3.17.0` or greater.
 
+:::info
+
+Note that this functionality was introduced in [v3.34.0](https://github.com/go-task/task/releases/tag/v3.34.0),
+so older versions are simply not yet able to give you this warning.
```

Comment:
```suggestion
so older versions are not able to give you this warning.
```

Nit: Avoid using terms like "simply" in docs. This is a habit I sometimes fall into too.

Also, "not yet" implies they might one day be able to which they won't.